output/output.go

```diff
@@ -6,40 +6,34 @@ import (
 	"sort"
 	"strings"
 
-	"github.com/graphaelli/hey/requester"
+	"github.com/simitt/hey/requester"
 )
 
-func PrintResults(work []*requester.Work, dur float64, writer io.Writer) {
-	for i, w := range work {
-		if i > 0 {
-			fmt.Fprintln(writer)
-		}
-
-		statusCodeDist := w.StatusCodes()
-		codes, total := SortedTotal(statusCodeDist)
-		div := float64(total)
-		fmt.Fprintln(writer, w.Request.URL, i)
-		for _, code := range codes {
-			cnt := statusCodeDist[code]
-			fmt.Fprintf(writer, "  [%d]\t%d responses (%.2f%%) \n", code, cnt, 100*float64(cnt)/div)
-		}
-		fmt.Fprintf(writer, "  total\t%d responses (%.2f rps)\n", total, div/dur)
+func PrintResults(w *requester.Work, dur float64, writer io.Writer) {
+	statusCodeDist := w.StatusCodes()
+	codes, total := SortedTotal(statusCodeDist)
+	div := float64(total)
+	fmt.Fprintln(writer)
```

Comment:
this just prints an empty line, you could also print the url like before, that you can pass to `PrintResults` from `target.Url`